app/src/main/java/mihon/core/migration/Migration.kt

```diff
@@ -0,0 +1,19 @@
+package mihon.core.migration
+
+interface Migration {
+    val version: Float
```

Comment:
why float?

Reply:
To allow multiple migrations for the same version and allow to choose if it should run after another

Reply:
Gigabrain